spec/griddler/email_spec.rb

```diff
@@ -289,6 +289,19 @@
     expect(body_from_email(text: body)).to eq 'Hello.'
   end
 
+  it 'should trim signature with non-breaking space after hyphens' do
+    body = <<-EOF
+      Hello.
+
+      --\xC2\xA0
+      Mr. Smith
+      CEO, company
+      t: 6174821300
+    EOF
+
+    expect(body_from_email(text: body)).to eq 'Hello.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.